test/test_linear_physical_object.py

```diff
@@ -22,46 +23,50 @@ def setup(self):
         self.position_meter = Vec2d(10, 50)
         self.position_pixel = meter_to_pixel(Vec2d(self.position_meter))
         self.velocity = Vec2d(16, 50)
+        self.external_forces = [CommonConstants.ROCKET_MASS * Vec2d(-10, 2)]
         self.acceleration = Vec2d(-10, 2)
-        self.mass = 1e5
+
+        self.mass = CommonConstants.ROCKET_MASS
 
     def test_init(self, example_image):
-        obj = LinearPhysicalObject(
+        _ = LinearPhysicalObject(
             image=example_image,
             pos=self.position_pixel,
             mass=self.mass,
             velocity=self.velocity,
-            acceleration=self.acceleration,
+            external_forces=self.external_forces,
         )
-        obj = LinearPhysicalObject(
+        _ = LinearPhysicalObject(
             image=example_image,
             pos=self.position_pixel,
             mass=self.mass,
             velocity=self.velocity,
         )
-        obj = LinearPhysicalObject(
+        _ = LinearPhysicalObject(
             image=example_image,
             pos=self.position_pixel,
             mass=self.mass,
         )
 
     def test_step(self, example_image):
+        # Given
```

Comment:
I think i read something about a way to run tests with different parameters. Maybe this could be useful instead of the "given/when/then" structure. 
Not necessary to change now, but in the rare event that your wisdom is wrapping around this particular bit of knowledge, maybe this could be a way to do it in the future. Or maybe not. I don't remember exactly how and when parametrical tesing is implemented best.

Reply:
parametrized tests are indeed a thing, but I would not see a typical use case here.